pkg/volume/awsebs/aws_ebs.go

```diff
@@ -263,6 +263,23 @@ func (plugin *awsElasticBlockStorePlugin) ConstructVolumeSpec(volName, mountPath
 	if err != nil {
 		return nil, err
 	}
+	volumeID, err = formatVolumeID(volName, volumeID)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to get AWS volume id from mount path %q: %v", mountPath, err)
+	}
+
+	awsVolume := &v1.Volume{
```

Comment:
Since the PV is almost the same as the block PV, can we move it into a common function with `volumeMode` as a param?